src/c20_client/client.py

```diff
@@ -41,36 +43,72 @@ def get_result_for_job(job, api_key):
     job_type = job['job_type']
 
     if job_type == 'documents':
-        print("Getting documents from regulations.gov...\n")
-        data = get_documents(
-            api_key,
-            job["page_offset"],
-            job["start_date"],
-            job["end_date"])
-        results = package_documents(data, CLIENT_ID, job_id)
+        results = find_documents_data(api_key, job, job_id)
 
     elif job_type == 'document':
-        print("Getting document from regulations.gov...\n")
-        data = download_document(
-            api_key,
-            job['document_id']
-        )
-        results = package_document(data, CLIENT_ID, job_id)
+        results = find_document_data(api_key, job, job_id)
 
     elif job_type == 'docket':
-        print("Getting docket from regulations.gov...\n")
-        data = get_docket(
-            api_key,
-            job['docket_id']
-        )
-        results = package_docket(data, CLIENT_ID, job_id)
+        results = find_docket_data(api_key, job, job_id)
+
+    elif job_type == 'download':
+        results = find_download_data(api_key, job, job_id)
+
+    elif job_type == 'none':
+        results = CLIENT_ID, job_id
```

Comment:
This will cause the client to call `return_result`, but it should not.  I suggest you make this condition first and simply return (there really is no work to do).